codecov.yml

```
@@ -1,2 +1,8 @@
 comment:
   after_n_builds: 4
+
+codecov:
+  disable_default_path_fixes: true # manual fix properly covers all files
```

```suggestion
  disable_default_path_fixes: true # Automatic detection does not discover all files
```

The comment isn't clear to me